go: v1.5.3 -> v1.5.4

rushmorem opened the pull request.

rushmorem committed:
go: v1.5.3 -> v1.5.4

vcunat commented:
I wonder if that would also fix the test errors from a couple of days ago http://hydra.nixos.org/job/nixpkgs/trunk/go_1_5.x86_64-linux

rushmorem commented:
Yes I noticed that. It's what motivated me look into this. If it's still broken after the update I will debug and fix.

rushmorem committed:
go: make 1.6 default

rushmorem committed:
storebrowse: mark broken

`storebrowse` depends on https://code.google.com/archive/p/gosqlite/
which leads to gosqlite.googlecode.com/hg/sqlite which now 404s.

rushmorem commented:
@viric do you still want to keep `storebrowse` around?
